src/contract/ContractFunctionResult.ts

```diff
@@ -65,7 +65,11 @@ export class ContractFunctionResult {
         return this.bytes;
     }
 
-    public getString(index: number): string {
+    public getString(index?: number): string {
```

Comment:
We should make this change for all accessors

Reply:
Should this be spun out as an issue, or should it be included in this pr?

Reply:
this PR, we shouldn't merge in a one-off change like this